### ISSUE TITLE
How to install the modules

### ISSUE THREAD

**Issue by Edin72** [Aepelzen/AepelzensModules#10]
I'm struggling with installing the modules. Simply download of the zip file and unzip in a rack folder doesn't work. I can't see the modules after restart.

**Comment by Aepelzen** [comment]
Hi. You can't just extract the zip. This repository only contains the sourcecode. To use the modules you have to compile them into a plugin first.

There are [instructions](https://github.com/VCVRack/Rack#building-plugins) on how to do that in the rack repository (i think there are also some more detailed instructions in the wiki).

Note: this hasn't been updated to 0.6 yet so you have to build it against the 0.5 branch of rack or wait a couple of days until i update this

**Comment by Edin72** [comment]
Hi
Many thanks for your reply. I have read the instructions and tried but without success. After hours of installing of java SRE, Apache Ant, MSYS2..., Eclipse...I gave up..I'm just not into it..I just use VCV as it is.
Would there be a compiled plugin available for download?

**Comment by Aepelzen** [comment]
You don't need any of those except for msys2. Have you checked [this](https://github.com/VCVRack/Rack#setting-up-your-development-environment) ?

Currently there are no compiled versions available. I'm not yet sure if I will provide one at some point, but even if I do it will be Linux only. But the Rack community team seems to be planning to provide binaries for the plugin manager, so maybe they will release a build at some point.

**Comment by Edin72** [comment]
Yes I have but there was a problem with building with the "make dep"...it is not recognizable command. After that I couldn't go further.

**Comment by benirose** [comment]
I was able to build for mac against v0.5, I'm happy to contribute the binary, but I'm not 100% sure how.

**Comment by Atavic** [comment]
Just share it !

**Comment by Aepelzen** [comment]
benirose: Thanks for the offer. There is already a mac binary posted in another issue but I'm not sure which version that was built off (judging by the date I would assume it is the latest).
Could you check that one first and just add your build to that same thread if it is newer?

**Comment by xyybob** [comment]
How do I install/build/make work the libsndfile dep? I've installed from MegaNerd but I'm not sure what to do with it now as make process doesn't find it. Pointers would be much appreciated here

**Comment by veryfungi** [comment]
I would love to have Burst for windows. I would also need a compiled version. :)

**Comment by audinf** [comment]
Yes please. I also would love to have these compiled. Having them added to the VCV Rack Plugin Manager would even be better.
